tests/commons/j2/test_j2_functions.py

```
@@ -30,10 +30,13 @@ def assertFormat(
     )
 
 
-def assertCase(short_name: str, case: tuple) -> None:
+def assertCase(
+    short_name: str, case: tuple, pfx_variants: list[str] | None = ["xsd:"]
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)